### PR TITLE
Dashboard code cleanup

### PR DESCRIPTION
**Pull request by EUdds**
Basically ran everything through prettier, commented everything, and added headers to the JS files so new members we can easily be introduced to the code.

### CODE REVIEW

File: dashboard/constants.js [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
-     }  
-}
\ No newline at end of file
+/*
+Author: Eric Udlis
+Purpose: Store all constants in one place
+*/
+module.exports = {
+  serverAddr: {
+    ip: "127.0.0.1",
```
Comment (rdaruwala) [inline dashboard/constants.js]: Why'd we change this back to 127.0.0.1? Do your internal tests still work if we have it listening on all interfaces?

File: dashboard/server.js [hunk 2]
```diff
@@ -1,88 +1,79 @@
-const constants = require('./constants');
-const DATA_SEND_RATE = 30;
-const dgram = require('dgram');
-const IP = constants.serverAddr.ip, PORT = constants.serverAddr.port; 
-
-/* We're gonna come up with a damn template for sending data from the server to the dashboard one day
-*
-*
-* I think today is that day
+/*
```
Comment (rdaruwala) [inline dashboard/server.js]: Optional, but consider making a "tests" folder or something like for this + other files.

With a filename such as "server.js" I'm inclined to think that this is the file that actually runs the dashboard server

File: dashboard/constants.js (hunk 1 above)
Comment (EUdds) [reply dashboard/constants.js]: Oops, didn't notice that, 127.0.0.1 or 0.0.0.0 works for my internal tests so I guess I just didn't notice

File: dashboard/server.js (hunk 2 above)
Comment (EUdds) [reply dashboard/server.js]: 👍

**Review by rdaruwala — APPROVED** [review]
Looks good!

**Pull request merged**